lib/public/AppFramework/Utility/ITimeFactory.php

```diff
@@ -58,4 +58,12 @@ public function getDateTime(string $time = 'now', \DateTimeZone $timezone = null
 	 * @since 26.0.0
 	 */
 	public function withTimeZone(\DateTimeZone $timezone): static;
+
+	/**
+	 * @param string|null $timezone
+	 * @return \DateTimeZone
```

Comment:
```suggestion
	 * @return \DateTimeZone Requested timezone if provided, UTC otherwise
```
You need to document what passing null does. Looking at the implementation it seems to always return UTC?

Reply:
> Looking at the implementation it seems to always return UTC?

Only if `withTimeZone()` was not called?

Reply:
Indeed. So my suggestion is not the right one, but there should be an explanation as to what passing null means.